pkg/operator/operator.go

```diff
@@ -219,22 +219,24 @@ func (c *vSphereProblemDetectorController) runNodeChecks(checkContext *check.Che
 		return nil, err
 	}
 
-	// Prepare list of errors of each check
+	// Prepare list of errors of each nodeCheck
 	checkErrors := make(map[string][]error)
```

Comment:
we should either used some named types or describe `make(map[string][]error` and what does this store.

Reply:
done